front-end/src/components/product/ProductItem.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import {  defineProps } from 'vue';
+ 
+    const props = defineProps({
+      product: {
+        type: Object,
+        required: true,
+      }
+    });
+</script>
+
+<template>
+    <div class="itemProducts">
+        <a :href="`/products/${props.product.id}`">
+            <span class="itemProducts_label"></span>
+            <span class="imgPrd">
+                <img :src=props.product.image 
+                    :alt=props.product.name
+                    style="background: 
+                    url(&quot;https://cong-news.appwifi.com/wp-content/uploads/2023/02/cốt-dừa-đậu-xanh_1.jpg&quot;) 
```

Comment:
Mấy hình này a download về dự án rồi dùng nha

Reply:
Cái này là data demo thôi mà em. mấy hình này sau này lưu bên BE chứ đúng ko ta? a tải về thì lưu kiểu ntnao hè? bth là lưu trong cái assets của FE còn BE a chưa rõ lắm. thôi để tạm v fix sau đc ko?